src/main/java/org/folio/circulation/domain/OpeningPeriod.java

```diff
@@ -0,0 +1,46 @@
+package org.folio.circulation.domain;
+
+import io.vertx.core.json.JsonObject;
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+import org.joda.time.LocalDate;
+import org.joda.time.LocalTime;
+import org.joda.time.format.DateTimeFormat;
+import org.joda.time.format.DateTimeFormatter;
+
+public class OpeningPeriod {
+  private static final String OPENING_DAY_KEY = "openingDay";
+  private static final String DATE_KEY = "date";
+  private static final String DATE_TIME_FORMAT = "yyyy-MM-dd'T'HH:mm:ss.SSSZ";
+  private static final DateTimeFormatter DATE_TIME_FORMATTER =
```

Comment:
Can we use `ISODateTimeFormat.dateTime().withZoneUTC()` here?

Reply:
yes, fixed